### PR TITLE
Add Azure Pipelines

### PR DESCRIPTION
**Pull request by Ethan-Arrowood**
Removes Travis and adds Azure Pipelines

### CODE REVIEW

**Comment by Ethan-Arrowood** [comment]
@mcollina I can't seem to create the new pipeline on Azure. When I go to fastify pipelines and click 'New' then 'New build pipeline', a list comes up with all the repos I have on my GitHub profile; however, none of the fastify ones are on the list (only my forks)

**Comment by mcollina** [comment]
Can you push this again? I enabled it (see #53).

File: azure-pipelines.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+steps:
+- task: NodeTool@0
+  inputs:
+    versionSpec: $(node_version)
+  displayName: Install Node.js
+
+- task: Npm@1
+  inputs:
+    command: custom
+    customCommand: 'ci'
```
[the remark this answers is not in the document]
Comment (Ethan-Arrowood) [reply azure-pipelines.yml]: Ahh yes you're right, `npm ci` does require a package-lock. Ill change this to standard `npm install`

**Comment by mcollina** [comment]
Badges in the README will need to be updated as well.

**Comment by Ethan-Arrowood** [comment]
Am i good to merge? I wouldn't create a new release for this would I?

**Pull request merged**